cmd/legacy/daemon.go

```diff
@@ -315,6 +307,17 @@ func (d *Daemon) Start() error {
 	go controllerMgr.Start(ctx)
 	mainLogger.Info("Started controller manager")
 
+	//Set health checks
```

Comment:
Small style nit. Leading spaces after the `//` are preferred, unless there's some kind of pragma that the comment is trying to invoke (like `//go:build`, for example)
```suggestion
	// Set health checks
```